python/MooseDocs/extensions/package.py

```diff
@@ -115,7 +115,7 @@ def defaultSettings():
 
     def createToken(self, parent, info, page):
         content = info['inline'] if 'inline' in info else info['block']
-        content = re.sub(r'__(?P<package>[A-Z]+)__', self.subFunction, content, flags=re.UNICODE)
+        content = re.sub(r'__(?P<package>[A-Z\_]+)__', self.subFunction, content, flags=re.UNICODE)
```

Comment:
You don't have to escape characters in character classes. In fact this might be slightly incorrect now because I believe it'll match A-Z and underscore and a backslash.

Reply:
I don't know why I "escaped" the underscore, I don't think it is ever needed.